assets/css/style.css

```diff
@@ -248,4 +251,186 @@ body {
 .display-data-p {
     margin-left: 5%;
     font-style: oblique;
-}
\ No newline at end of file
+}
+
+@media (max-width: 1860px) {
+    .display-data {
+        width: 350px;
+    }
+}
+
+@media (max-width: 1735px) {
+    .display-data {
+        width: 330px;
+    }
+}
+
+@media (max-width: 1655px) {
+    .display-data {
+        width: 310px;
+    }
+}
+
+@media (max-width: 1575px) {
```

Comment:
This values should be standard ones, for responsive there are standard values like 768p or like from this images.:
![image](https://user-images.githubusercontent.com/15923693/112019771-0ac05500-8b30-11eb-8c7b-96531a5bd644.png)
so refactor it to be standard breakpoints, on all places.

Reply:
Yes, okay. I've just tried to use media-queries wherever change was needed and that is why I had so many of them, but okay, i can do it only for standard ones.